src/main/webapp/WEB-INF/Fieldbook-servlet.xml

```diff
@@ -108,8 +108,6 @@
 
 	<!-- This bean is singleton (default) scoped because there nothing that it represents which is request scoped. -->
 	<bean id="fieldbookProperties" class="com.efficio.fieldbook.web.util.FieldbookProperties">
-		<property name="programLocationsUrl" value="${url.program.locations}" />
-		<property name="programBreedingMethodsUrl" value="${url.program.breeding.methods}" />
```

Comment:
can you remove it from fieldbook.properties too?

Reply:
@nahuel-soldevilla it's done.